src/libraries/System.Text.Json/src/System/Text/Json/Nodes/JsonValue.CreateOverloads.cs

```diff
@@ -225,7 +225,7 @@ public partial class JsonValue
         /// <param name="options">Options to control the behavior.</param>
         /// <returns>The new instance of the <see cref="JsonValue"/> class that contains the specified value.</returns>
         [return: NotNullIfNotNull(nameof(value))]
-        public static JsonValue? Create(string? value, JsonNodeOptions? options = null) => value != null ? new JsonValueTrimmable<string>(value, JsonMetadataServices.StringConverter) : null;
+        public static JsonValue? Create(string? value, JsonNodeOptions? options = null) => value != null ? new JsonValueTrimmable<string?>(value, JsonMetadataServices.StringConverter) : null;
```

Comment:
Not sure I understand why this change was made.

Reply:
Previously it was ok with `JsonValueTrimmable<string>` because even though `value` is a `string?`, this ctor is only used after `value != null`.  But now `StringConverter` uses type `string?` rather than `string`, so the `T` needs to be `string?`.